src/lib/discord/commands/_repositories.ts

```diff
@@ -1,23 +1,93 @@
-/**
- * @TODO replace with GitHub API call
- */
+import { getRepos } from '../../github/queries'
+
+const nameToAlias = new Map<string, string>([
+  ['amplify-cli', 'cli'],
+  ['amplify-ios', 'ios'],
+  ['discord-bot', 'discord-bot'],
+  ['amplify-ui', 'ui'],
+  ['amplify-js', 'js'],
+  ['docs', 'docs'],
+  ['amplify-android', 'android'],
+  ['amplify-flutter', 'flutter'],
+  ['amplify-category-api', 'category-api'],
+  ['amplify-cli-export-construct', 'cli-export-construct'],
+  ['amplify-codegen', 'codegen'],
+  ['amplify-adminui', 'adminui'],
+  ['amplify-hosting', 'hosting'],
+  ['Test-Repository', 'TEST'],
```

Comment:
is this needed for testing?

Reply:
Yeah I'm still using my test app installation which only has the Test-Repository, so without this here no options will show up for the `<repo>` option. But I'll take it out for the pr.